xarray/core/ops.py

```diff
@@ -198,6 +198,24 @@ def func(self, *args, **kwargs):
     return func
 
 
+def rolling_count(rolling):
```

Comment:
The problem with applying a `move_sum` on the `not_nulls` is that there are no `null`s left in `not_nulls` (they are `False`), and so `min_count` won't work correctly. 

But It does feel like this is a bit awkward atm, very open to other suggestions